### ISSUE TITLE
Add repository for Ubuntu 21.10

### ISSUE THREAD

**Issue by fakhamatia** [phpmyadmin/phpmyadmin#17345]
Please Add repository for Ubuntu 21.10 Impish Indri 

```
$ sudo add-apt-repository ppa:phpmyadmin/ppa
Repository: 'deb http://ppa.launchpad.net/phpmyadmin/ppa/ubuntu/ impish main'
Description:
Test packages for Ubuntu, maintained on salsa.debian.org/phpmyadmin-team

Do not use these packages on production! Wait until they are released through the official ubuntu-channels.
More info: https://launchpad.net/~phpmyadmin/+archive/ubuntu/ppa
Adding repository.
Press [ENTER] to continue or Ctrl-c to cancel.
Adding deb entry to /etc/apt/sources.list.d/phpmyadmin-ubuntu-ppa-impish.list
Adding disabled deb-src entry to /etc/apt/sources.list.d/phpmyadmin-ubuntu-ppa-impish.list
Adding key to /etc/apt/trusted.gpg.d/phpmyadmin-ubuntu-ppa.gpg with fingerprint 
Err:22 http://ppa.launchpad.net/phpmyadmin/ppa/ubuntu impish Release
  404  Not Found [IP: 91.189.95.85 80]
Reading package lists... Done               
E: The repository 'http://ppa.launchpad.net/phpmyadmin/ppa/ubuntu impish Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Comment by ibennetch** [comment]
Thanks for the report.

Assigned to @williamdes.

**Comment by williamdes** [comment]
I am holding the release into Debian and Ubuntu until we release 5.1.3, so that should not be in a long time

Thank you for the report

Everything is packaged and ready, just waiting for the latest phpMyAdmin fixes (#17308 is the one I really want to have included)

**Comment by williamdes** [comment]
Hi @fakhamatia !
I built and released the version 🎉 
And wrote a test script to test builds using Docker: https://github.com/phpmyadmin/phpmyadmin/wiki/DebianUbuntu#testing-the-ubuntu-ppa-package-with-docker
